Documentation/ApiOverview/Xclasses/Index.rst

```diff
@@ -59,7 +59,7 @@ Limitations
   and has to be maintained and fixed if the underlying code changes.
   If possible, you should use a hook instead of an XCLASS.
 
-- XCLASSes do **not** work for static classes, static methods or final classes.
+- XCLASSes do **not** work for static classes, static methods, abstract classes or final classes.
```

Comment:
Is that so? Where do you have the information from? Did you try it out?

Reply:
See original PR #3755.